log.c

```diff
@@ -181,8 +181,9 @@ log_verbose_reset(void)
 
 	for (i = 0; i < nlog_verbose; i++)
 		free(log_verbose[i]);
-	free(log_verbose);
 	log_verbose = NULL;
+	free(log_verbose);
```

Comment:
free(NULL) is a no-op so it won't blow up but this adds a memory leak of log_verbose